site/content/entry/citra-progress-report-2023-q1/index.md

```diff
@@ -0,0 +1,413 @@
+
+In the past, Citra faced a problem where any copies or operations held at the end of the emulated VRAM were not understood correctly, causing them to be completely ignored. This would cause numerous errors in the emulation of games, including some graphics being cut out entirely on HarmoKnight. 
+
+However, by allowing MemoryRef, a wrapper over a pointer to physical memory used for serialization, these values can now be correctly understood and executed. This means that the errors previously experienced due to ignored values will be resolved, leading to a more accurate and improved emulation of games on Citra.
+
+{{< sidebyside "image" ""
+    "harmoOld.png=What's happened there?"
+    "harmoNew.png=Ah, that's better!" >}}
+
+### arm/dynarmic: More accurate cycle counting ([#6194](https://github.com/citra-emu/citra/pull/6194)) by [merryhime](https://github.com/merryhime)
+
+While most games easily run at full speed nowadays, a long standing issue in Citra has been pre-rendered cutscene videos running way too slowly. This has always been a bit of a mystery, since decoding a video at a resolution of 240p shouldn’t be too taxing on the system of an average user.
+
+By implementing more accurate cycle counting, we are happy to announce we have made the first step towards solving this problem, improving FMV video performance by up to 3x! To understand what this means, we need to take a closer look at how a processor works.
+
+On the lowest level, all that a CPU does is process a list of instructions. Some instructions however take longer to execute than others. The time it takes for an instruction to execute is measured in cycles. Previously, we were using a very naive implementation of assuming every instruction would take one cycle. This leads to games idling longer than they should, therefore unnecessarily slowing down the emulation.
+Now [merryhime](https://github.com/merryhime), the author of [dynarmic](https://github.com/merryhime/dynarmic), has implemented much more accurate cycle counts, significantly reducing this issue. Performance is still problematic due to currently unknown reasons, so please expect FMV videos to run at half speed on hardware that only meets our minimum recommendations.
+
+## Multiplayer Fixes
+
+### APT: implement Set and GetWirelessRebootInfo ([#5328](https://github.com/citra-emu/citra/pull/5328)) by [B3n30](https://github.com/B3n30)
+
+This PR implements two key services relating to Download Play Applet functionality on Citra. `SetWirelessRebootInfo` and `GetWirelessRebootInfo` are used to transfer information and arguments between reboots of Download Play. These are necessary so that the app is able to start another app, like in the way Download Play works via transferring a game demo from one console to another. Download Play does not work in Citra just yet. This PR is just to law the groundwork for it to hopefully work in the future!
```

Comment:
```suggestion
This PR implements two key services relating to Download Play Applet functionality on Citra. `SetWirelessRebootInfo` and `GetWirelessRebootInfo` are used to transfer information and arguments between reboots of Download Play. These are necessary so that the app is able to start another app, like in the way Download Play works via transferring a game demo from one console to another. Download Play does not work in Citra just yet. This PR is just to lay the groundwork for it to hopefully work in the future!
```